plugins/com.google.cloud.tools.eclipse.appengine.localserver.test/src/com/google/cloud/tools/eclipse/appengine/localserver/ServiceAccountUtilTest.java

```diff
@@ -64,7 +65,9 @@ public void setUp() throws IOException {
     Iam iam = mock(Iam.class);
     Projects projects = mock(Projects.class);
     ServiceAccounts serviceAccounts = mock(ServiceAccounts.class);
-        
+    
+    when(apiFactory.hasCredentialsSet()).thenReturn(true);
```

Comment:
When the api factory has to instantiate clients, we just follow the mockito approach.